### PR TITLE
add unit test for Balance opcode

### PR DESCRIPTION
**Pull request by ChengYueJia**

### CODE REVIEW

File: bus-mapping/src/evm/opcodes/balance.rs [hunk 1]
```diff
@@ -43,3 +43,121 @@ impl Opcode for Balance {
...
+    };
+    use mock::eth;
+    use mock::test_ctx::{helpers::*, TestContext};
+    use pretty_assertions::assert_eq;
+
+    /// EVM behaviour
+    /// The BALANCE opcode pops address (20 bytes of data) off the stack
+    ///    * And pushes the balance of the corresponding account onto the stack.
+    ///    * If the given account doesn't exist (by checking non existing flag),
+    ///      then it will push 0 onto the stack instead.
```
Comment (CPerezz) [inline bus-mapping/src/evm/opcodes/balance.rs:68]: It's not required to explain the opcode functioning here. You can add small comments inside of the test if you're specifically checking something mentioned in these cases.

**Review by CPerezz — APPROVED** [review]
Thanks for the work.
It's always nice to improve the test coverage!

File: bus-mapping/src/evm/opcodes/balance.rs (hunk 1 above)
Comment (ChengYueJia) [reply bus-mapping/src/evm/opcodes/balance.rs]: Thanks for pointing it out. It's done

**Comment by CPerezz** [comment]
Thanks for fixing @ChengYueJia 

Will merge as soon as the Integration tests finally are picked up by the worker and pass!!

**Pull request merged**